packages/web-components/fast-components/src/menu-item/menu-item.definition.ts

```diff
@@ -56,6 +56,10 @@ export const fastMenuItemDefinition: WebComponentDefinition = {
                     name: "end",
                     description: "The end slot",
                 },
+                {
+                    name: "submenu",
+                    description: "The submenu slot",
+                },
```

Comment:
I think when you update/rebase the branch - this file will need to include title as well as description